### DISCUSSION TITLE
Could not initialize interactsh client

### DISCUSSION THREAD

**Discussion by onurdrsn** [projectdiscovery/nuclei#2039] [Q&A] [answered]
**Describe the bug?**
My Nuclei Current Version: 2.5.7
but for some reason I am getting such error **"Could not initialize interactsh client"**. What is the solution?
My command when running nuclei is:
      
    └─# nuclei --version  nuclei -l test -t cves/2022 

                       __     _
     ____  __  _______/ /__  (_)
    / __ \/ / / / ___/ / _ \/ /
    / / / / /_/ / /__/ /  __/ /
    /_/ /_/\__,_/\___/_/\___/_/   2.5.7
    
    projectdiscovery.io
    
    [WRN] Use with caution. You are responsible for your actions.
    [WRN] Developers assume no liability and are not responsible for any misuse or damage.
    [WRN] Found 4 templates with syntax warning (use -validate flag for further examination)
    [WRN] Found 4 templates with syntax error (use -validate flag for further examination)
    [WRN] Found 1 templates with runtime error (use -validate flag for further examination)
    [INF] Using Nuclei Engine 2.5.7 (outdated)
    [INF] Using Nuclei Templates 9.0.2 (latest)
    [INF] Using Interactsh Server https://interact.sh
    [INF] Templates added in last update: 3564
    [INF] Templates loaded for scan: 63
    [ERR] Could not initialize interactsh client: 
    could not create client: could not make register request: POST https://interact.sh/register giving up after 6 attempts: Post "https://interact.sh/register": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Comment by ehsandeep** [comment]
@onur55-tr as you can see

> [INF] Using Nuclei Engine 2.5.7 (outdated)


You are running outdated version of nuclei, update your nuclei using `nuclei -update` or `go install` to update to the latest one.

**Comment by onurdrsn** [comment, marked as answer]
Mmm... There is no `-update` flag, what is the equivalent instead? and when I run `go install` nothing is returned.

**Reply by ehsandeep** [reply]
```
go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest
```

or download the binary file from here - https://github.com/projectdiscovery/nuclei/releases/tag/v2.7.1

**Reply by onurdrsn** [reply]
Thx very interesting in version 2.7.1 downloaded from go but 2.5.7 in `/usr/bin`. This problem is solved, but there is another problem. I'll open it up for another discussion.
